src/main/java/com/farmtrade/controllers/UserController.java

```diff
@@ -0,0 +1,47 @@
+package com.farmtrade.controllers;
+
+import com.farmtrade.entities.User;
+import com.farmtrade.services.impl.UserServiceImpl;
+import org.springframework.data.domain.Page;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("users")
+public class UserController {
+    final private UserServiceImpl userServiceImpl;
+
+    public UserController(UserServiceImpl userServiceImpl) {
+        this.userServiceImpl = userServiceImpl;
+    }
+
+
+    @GetMapping("/all")
```

Comment:
`/all` should be remove. REST would like to see Page/List of users by `http:localhost:8080/users` path